css-two-column-layout/styles.css

```diff
@@ -0,0 +1,87 @@
+.container {
+  max-width: 1100px;
+  margin: auto;
+}
+
+.row {
+  display: flex;
+}
+
+.column-fourth {
+  flex-basis: calc(100% / 2);
+}
+
+.row2 {
+  display: flex;
+}
```

Comment:
You don't need a numbered variant of this class, just use the `row` class again.